### PR TITLE
Adding NCL_polar_1_lg example python file

### PR DESCRIPTION
**Pull request by michaelavs**
polar_1

### CODE REVIEW

**Comment by jukent** [comment]
@michaelavs Can you link the issue that your PR is addressing? 

In the main description you would write: Closes issue #58 
That way when the PR is merged it automatically closes the issue, and makes it easier for reviewers to compare the plot you produced and the target image. @clairefio @hCraker This is best practice for your PRs too.

**Comment by jukent** [comment]
My one concern is the latitudinal lines drop out between 150E and 180.

File: Plots/Contours/NCL_polar_1_lg.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import matplotlib.path as mpath
+import geocat.datafiles as gdf
+import matplotlib.ticker as mticker
+from geocat.viz import util as gvutil
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc"))
+U=ds.U[1,:,:]
+
+# Fix the artifact of not-shown-data around 0 and 360-degree longitudes
```
Comment (jukent) [inline Plots/Contours/NCL_polar_1_lg.py:22]: Perhaps we need a line similar to this for fixing the artifact of not-shown-latitude lines around 150E-180 longitudes. @erogluorhan Any insight on this?

Comment (erogluorhan) [reply Plots/Contours/NCL_polar_1_lg.py:22]: Will have a look at this soon

**Pull request closed without merging**